features/resolution-compat.yml

```diff
@@ -0,0 +1,8 @@
+name: resolution media query (compatibility prefixes)
```

Comment:
```suggestion
name: "@media resolution (compatibility prefixes)"
```

Reply:
"resolution media query (compatibility prefixes)" is modeled after `resolution.yml`'s name. If this changes (which is a reasonable suggestion), we'd also need to change the other feature.

(I'm a tiny bit worried about having a bunch of `@media` features but this doesn't strike me as a "web" prefix situation.)